transformer_engine/pytorch/csrc/ts_fp8_op.cpp

```diff
@@ -316,6 +318,11 @@ at::Tensor te_gemm_ts(at::Tensor A,
   bool accumulate_arg = static_cast<bool>(accumulate);
   bool use_split_accumulator_arg = static_cast<bool>(use_split_accumulator);
 
+  // Set an external SM Margin to all the GEMMs. This comes in handy when DP is overlapped with GEMMs
+  cudaDeviceProp prop;
+  cudaGetDeviceProperties(&prop, 0);
+  int num_math_sms = prop.multiProcessorCount - transformer_engine::getenv<int>("NVTE_EXT_MARGIN_SM", 0);
```

Comment:
This doesn't seem to be used. Do we need to pass it to `te_gemm`?

Reply:
Yes, I missed that in this branch, fixed